Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.12-alpine
 
 RUN --mount=source=.,target=/root/src/proxpi,rw \
     uname -a && cat /etc/issue && apk --version && python --version && pip --version \
- && apk --no-cache add git \
+ && apk --no-cache add git build-base libxslt-dev libxml2-dev \
```

Comment:
I had build failures related to the lxml library which expects gcc and these libraries to be available to build. They likely don't build arm wheels so lxml needs to build from source.